<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Traductor</class>
 <widget class="QDialog" name="Traductor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>489</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Traductor v.0.1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recoverdrake.qrc">
    <normaloff>:/Imagenes/traductor.png</normaloff>:/Imagenes/traductor.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pushButton_2">
     <property name="toolTip">
      <string>Nos vamos...</string>
     </property>
     <property name="statusTip">
      <string>Nos vamos...</string>
     </property>
     <property name="whatsThis">
      <string>Nos vamos...</string>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
     <property name="icon">
      <iconset resource="recoverdrake.qrc">
       <normaloff>:/Imagenes/puerta.png</normaloff>:/Imagenes/puerta.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="toolTip">
      <string>Traducir seleccion</string>
     </property>
     <property name="statusTip">
      <string>Traducir seleccion</string>
     </property>
     <property name="whatsThis">
      <string>Traducir seleccion</string>
     </property>
     <property name="text">
      <string>Traducir</string>
     </property>
     <property name="icon">
      <iconset resource="recoverdrake.qrc">
       <normaloff>:/Imagenes/traductor.png</normaloff>:/Imagenes/traductor.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="6">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Tipo de traducci√≥n</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>Texto</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>Archivo</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="radioButton_3">
          <property name="text">
           <string>Pagina Web</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="2" column="3" colspan="3">
      <widget class="QTextEdit" name="textEdit_2"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Idioma origen</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Idioma destino</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Limpiar datos</string>
     </property>
     <property name="icon">
      <iconset resource="recoverdrake.qrc">
       <normaloff>:/Imagenes/borrar.png</normaloff>:/Imagenes/borrar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recoverdrake.qrc"/>
 </resources>
 <connections/>
</ui>
